### DISCUSSION TITLE
Multiple watches shown in "watch" window wearos dropdown

### DISCUSSION THREAD

**Discussion by Npavkov** [j-kaltes/Juggluco#102] [Q&A]
I have a galaxywatch4 setup with juggluco connecting the watch to the libre2 sensor already. My question is ....How do I get a 2nd watch Galaxywatch5 to show up in the dropdown selection? I would like to test the galaxywatch5 operationally with juggluco. I intend of disconnecting the galaxywatch4 to do a test of the GW5 watch.....currently I can not get the GW5 to show up in dropdown selection on WearOS config screen.  Any Ideas?

**Comment by j-kaltes** [comment]
If the WearOS watch is connected to the phone according to the Samsung Galaxy Wearable app, it is also selectable in the ‘Spinner” at the top of left menu→Watch→”WearOS Config”. It is possible to connect multiple WearOS watches at the same time to one Juggluco app.
![multiplewatches](https://github.com/j-kaltes/Juggluco/assets/72013079/7bc789a4-837a-4085-a33d-cab45f8ddc83)

**Reply by Npavkov** [reply]
thank you
